cmd/aspect/root/root.go

```diff
@@ -47,7 +45,7 @@ func NewRootCmd(
 	streams ioutils.Streams,
 	pluginSystem system.PluginSystem,
 	defaultInteractive bool,
-) *cobra.Command {
+) (*cobra.Command, error) {
```

Comment:
This is an analogue to a constructor in OO. I don't expect a constructor to return an error.